src/instance.ts

```diff
@@ -43,12 +43,12 @@ export default class Instance {
 
   constructor(options: InstanceOptions) {
 
-    if (!options.instanceId) throw new Error('Expected `instanceId` property in Instance options!');
-    if (options.instanceId.split(":").length !== 3) throw new Error('The `instanceId` property is in the wrong format!');
```

Comment:
Similar feedback to the Ruby lib - those errors might propagate through the stack. Devs using service SDKs will not understand those messages, as they don't use the instance class directly.